validation/exclusions_test.go

```diff
@@ -61,6 +61,7 @@ var exclusions = map[string]map[uint64]bool{
 		999999999: true, // sepolia/zora         MCP (at time of writing)
 		11155421:  true, // sepolia-dev0/oplabs-devnet-0
 		11763072:  true, // sepolia-dev0/base-devnet-0
+		11155420:  true, // sepolia/op (contract versions upgraded with gov approval pending)
```

Comment:
This is causing unit test failures, because we have a "meta test" which prevents us making exclusions for standard chains. We most likely need to take another look at how we want to deal with this -- but I think we can revert this change and fix the problems separately (it shouldn't block this PR).